src/rnanorm/methods/between_sample.py

```diff
@@ -0,0 +1,417 @@
+"""Between sample normalizations."""
```

Comment:
The file is long and collapsed by Github in code review. Should'd we split the functionality in more files? I'd prefer to have each method in a dedicated file, even though some implementations are short. It's also easier to track bug fixes and new implementations.